docs/examples/spectrum/plot_standard_ASTM_G173-03.py

```diff
@@ -0,0 +1,31 @@
+"""
+ASTM G173-03 Standard Spectrum
+==============================
+
+This example demonstrates how to read the data from the ASTM G173-03 standard
+spectrum bundled with pvlib and plot each of the components.
+
+The ASTM G173-03 standard provides reference solar spectral irradiance data.
+"""
+
+import matplotlib.pyplot as plt
+import pvlib
+
+
+# %%
+# Use :py:func:`pvlib.spectrum.get_standard_spectrum` to retrieve a
+# standard spectra file.
+
+am15 = pvlib.spectrum.get_standard_spectrum(standard="ASTM G173-03")
+
+# Plot
+plt.plot(am15.index, am15['extraterrestrial'], label='Extraterrestrial')
+plt.plot(am15.index, am15['global'], label='Global')
+plt.plot(am15.index, am15['direct'], label='Direct')
+plt.xlabel(r'Wavelength $[nm]$')
+plt.ylabel(r'Irradiance $[\frac{W}{m^2 nm}$')
```

Comment:
```suggestion
plt.ylabel(r'Irradiance $[\frac{W}{m^2 nm}]$')
```

Reply:
You could also try out using `\left[` and `\right]` to make it render a bit nicer, but I'm not sure it works in matplotlib's latex renderer.